awscli/customizations/ecs/executecommand.py

```diff
@@ -83,7 +84,7 @@ def invoke(self, service_name, operation_name, parameters, parsed_globals):
             # before calling execute-command to ensure that
             # session-manager-plugin is installed
             # before execute-command-command is made
-            check_call(["session-manager-plugin"])
+            check_call(["session-manager-plugin"], stdout=open(os.devnull, 'wb'))
```

Comment:
What do you think about simplifying this to just use `subprocess.DEVNULL` (and importing `DEVNULL` from `subprocess` up at the top)?

```suggestion
            check_call(["session-manager-plugin"], stdout=DEVNULL)
```